independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/handlers/MatrixParamHandler.java

```diff
@@ -6,7 +6,7 @@
 public class MatrixParamHandler implements ServerRestHandler {
     @Override
     public void handle(ResteasyReactiveRequestContext requestContext) throws Exception {
-        if (requestContext.getPath().contains(";")) {
+        if (requestContext.getPath().indexOf(';') != -1) {
```

Comment:
Although this makes sense, this handler is rarely used

Reply:
Agree, it is a minor I have found by reading few flamegraphs these days, but it can be ignored if we don't care really...
Although I would like to understand why I have found you, given that it shows ups be rarely used

Reply:
We can certainly include it, there is absolutely no harm done!

And actually I was wrong before... The handler is always called, it's just that `if` statement is rarely true (as HTTP Matrix params are rarely used).